packages/driver/src/cross-origin/communicator.ts

```diff
@@ -191,8 +191,9 @@ export class SpecBridgeCommunicator extends EventEmitter {
       data = preprocessLogForSerialization(data as any)
     }
 
-    // If requested by the runner, preprocess the final snapshot before sending through postMessage() to attempt to serialize the DOM body of the snapshot.
-    if (FINAL_SNAPSHOT_EVENT === eventName) {
+    // If requested by the runner, preprocess the snapshot before sending through postMessage() to attempt to serialize the DOM body of the snapshot.
```

Comment:
nit: lets specify @packages/runner since the driver has a Runner class as well

Reply:
sorry this was confusing...more like, we should specific which runner. it seems like it's this runner:
- https://github.com/cypress-io/cypress/tree/develop/packages/app/src/runner

but our code base also has these runners: 
- https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/runner.ts
- https://github.com/cypress-io/cypress/tree/develop/packages/runner